src/tests/ftest/slurm_setup.py

```diff
@@ -37,10 +46,18 @@
     "systemctl restart slurmctld",
     "systemctl enable slurmctld"]
 
+SLURMCTLD_STARTUP_DEBUG = [
+    "cat /var/log/slurmctld.log",
+    "grep -v '^#' /etc/slurm/slurm.conf"]
```

Comment:
In case you still want to to see the commented section names:
```suggestion
    "grep -v '^#\w' /etc/slurm/slurm.conf"]
```